watch.go

```diff
@@ -180,7 +204,8 @@ func (fb *Firebase) Watch(notifications chan Event) error {
 
 				// TODO: handle
 			case "rules_debug":
-				log.Printf("Rules-Debug: %s\n", txt)
+				txt, _ := ioutil.ReadAll(resp.Body)
+				log.Printf("Rules-Debug: %s\n", string(txt))
```

Comment:
You are right. It should only read until an empty line. I will fix it.

█  Jacques

On 31 July 2016 at 21:21, Steven Berlanga <notifications@github.com> wrote:

> In watch.go
> <https://github.com/zabawaba99/firego/pull/50#discussion_r72909276>:
>
> > @@ -180,7 +204,8 @@ func (fb *Firebase) Watch(notifications chan Event) error {
> >
> >  				// TODO: handle
> >  			case "rules_debug":
> > -				log.Printf("Rules-Debug: %s\n", txt)
> > +				txt, _ := ioutil.ReadAll(resp.Body)
> > +				log.Printf("Rules-Debug: %s\n", string(txt))
>
> I'll go ahead and run a test for this. My fear with reading the entire
> response body is that it will read until it get's an EOF which may cause
> other messages to be missed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zabawaba99/firego/pull/50/files/91637d68c23bcd6761930b23882b18621b759365#r72909276>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAB2iItELjgeYLWwDhws0ZwpOvgwld0Rks5qbPXVgaJpZM4JYHTS>
> .
>